### PR TITLE
Initial stuff for slack and dynamodb connectors

### PR DESCRIPTION
**Pull request by simlay**

### CODE REVIEW

File: content/connectors/sinks/dynamodb.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  AWS_REGION: "us-west-2"
+  AWS_ACCESS_KEY_ID: ""
+  AWS_SECRET_ACCESS_KEY: ""
+```
+
+### Parameters
+
+* `table-name` - The name of the [dynamodb table name]
+* `column-names` - A comma separated list of the keys. **The first key is the partition key**
+* `column-types` - A comma separated list of the types for the keys.
```
Comment (ajhunyady) [inline content/connectors/sinks/dynamodb.md:35]: Is there a reference guide on `column types` allowed?

File: content/connectors/sinks/dynamodb.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+[AWS Region]: https://aws.amazon.com/about-aws/global-infrastructure/regions_az/
+[dynamodb table name]: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-dynamodb-table.html
+
+This configuration file is used together with the `fluvio connector create` command, like so:
+
+%copy first-line%
+```bash
+$ fluvio connector create --config=./dynamodb-connector.yml
+```
+
```
Comment (ajhunyady) [inline content/connectors/sinks/dynamodb.md:54]: Let's also add `fluvio connector list` show show status.

Comment (simlay) [reply content/connectors/sinks/dynamodb.md:54]: For the sake of consistency with the other connector pages, I think we shouldn't. The output bound to change the most.

File: content/connectors/sinks/dynamodb.md (hunk 1 above)
Comment (simlay) [reply content/connectors/sinks/dynamodb.md]: I added it.

**Review by ajhunyady — APPROVED** [review]
Awesome!

**Pull request merged**